action/protocol/account/transfer.go

```diff
@@ -66,6 +66,7 @@ func (p *Protocol) handleTransfer(ctx context.Context, tsf *action.Transfer, sm
 
 	if fCtx.FixGasAndNonceUpdate || tsf.Nonce() != 0 {
 		// update sender Nonce
+		sender.CheckConvertToZeroNonceType(tsf.Nonce())
```

Comment:
don't need a hard-fork flag here, b/c this would only happen after the hard-fork when the nonce of clean address is changed by `AdjustNonceForCleanAddress()` to 0
for now, the first nonce of a clean address remains == 1, so this does no harm